### ISSUE TITLE
Supernode  - St. James Town

### ISSUE THREAD

**Issue by darkdrgn2k** [tomeshnet/toronto-community-network#66]
<sup>_This initial comment is collaborative and open to modification by all._</sup>

## Task Summary

🎟️ **Re-ticketed from:** #
📅 **Due date:** ON HOLD
🎯 **Success criteria:** Deploy Super Node

Deploy super node along side Cisco
- Possibly not with cisco
- Possible link to Thornliff?

~~77 Howard St - Gateway Node~~
Blocked by building admin

Other Possible candidates
- 135 Rose 
- 670 Parliament 
- 99 Howard

## To Do

- [ ] Design Super Node
- [ ] Pay for Super Node
- [ ] Deploy Super Node

**Comment by darkdrgn2k** [comment]
@tomeshnet/project-operations Can we reach out to the individuals who are interested in St James Town to see what support we can rally
